x/blob/keeper/keeper.go

```diff
@@ -81,7 +81,10 @@ func (k Keeper) UpdateBlobParams(goCtx context.Context, msg *types.MsgUpdateBlob
 
 	// Emit an event indicating successful parameter update.
 	if err := ctx.EventManager().EmitTypedEvent(
-		types.NewUpdateBlobParamsEvent(msg.Authority, msg.Params),
```

Comment:
oh huh, I'm not sure how Rachid and I missed this usage.

Reply:
Aaah, it's not used at all. The other one could be used by consumers in implementations like:
```
for _, e := range res.TxResult.GetEvents() {
				if e.Type == types.EventTypePayForBlob {
					signer := e.GetAttributes()[0].GetValue()
					_, err = sdk.AccAddressFromBech32(signer)
					require.NoError(err)
					blob, err := hex.DecodeString(tc.args[1])
					require.NoError(err)
					blobSize, err := strconv.ParseInt(e.GetAttributes()[1].GetValue(), 10, 64)
					require.NoError(err)
					require.Equal(len(blob), int(blobSize))
				}
			}
```

But since there is no test for this one, it's not used